lib/sedna/algorithms/hard_example_mining/__init__.py

```diff
@@ -56,6 +56,8 @@ def __call__(self, infer_result=None):
 
         image_score = 0
         for bbox in infer_result:
+            if len(bbox) < 5:
```

Comment:
Which case would this be? 
If so, then should we also ignore the entry for infer_result?